iodata/formats/wfx.py

```diff
@@ -109,6 +109,36 @@ def load_data_wfx(lit: LineIterator) -> dict:
         else:
             warnings.warn("Not recognized label, skip {0}".format(key))
 
+    # check required properties in WFX
+    required_prop = {'<Title>': 'title',
```

Comment:
The items listed in `required_prop` dictionary already exists in `labels_*` dictionaries, so I think this is a redundant dictionary. Also, only checking section tags is enough, so there is no need to use `prop_name` as well.